lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -2894,6 +2896,7 @@ ThreadContext::ClearInlineCaches()
     registeredInlineCacheCount = 0;
     unregisteredInlineCacheCount = 0;
 }
+#endif //PERSISTENT_INLINE_CACHES
```

Comment:
Should we just move this #endif to above ThreadContext::ClearInvalidatedUniqueGuards ?